.pre-commit-config.yaml

```diff
@@ -16,11 +22,6 @@ repos:
   - id: requirements-txt-fixer
   - id: mixed-line-ending
     args: ['--fix=no']
-  - id: flake8
-    args: ['--ignore=E203,E501,F811']  # default of Black
+  # - id: flake8
```

Comment:
Add back flake8, and for problematic files either disable it locally or entire files (https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors). If there's any formatting error where it disagrees with black, then add the code to this list.

Reply:
These are sorted now